sdk/metric/minmaxsumcount_stress_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019, OpenTelemetry Authors
```

Comment:
```suggestion
// Copyright 2020, OpenTelemetry Authors
```

Reply:
I'll do that (and the goroutine leak) in a separate PR

Reply:
Created PR #547 relating to copyright date and goroutine leak in histogram stress test.